components/table/README.md

```diff
@@ -522,3 +523,12 @@ The `d2l-table-controls` component can be placed in the `d2l-table-wrapper`'s `c
 | `no-sticky` | Boolean | Disables sticky positioning for the controls |
 | `select-all-pages-allowed` | Boolean | Whether all pages can be selected |
 <!-- docs: end hidden content -->
+
+## Accessibility
+
+The `d2l-table-wrapper` simply wraps an HTML table in order to add styles and functionality to that table. As such, it is important to use proper table markup. Recommendations are available in [this tutorial](https://www.w3.org/WAI/tutorials/tables/). As mentioned in the Best Practices section of this document, it is important to specify column and row headings so data is meaningful to screen reader users.
+
+The `d2l-table-col-sort-button` component was built with screen reader users at front-of-mind. Important features of note include:
+- A `source-type` attribute which lets users specify the type of data in the column in order to provide a more descriptive title to both screen reader and sighted users
+- Best effort to announce changes (e.g., change in sort order and/or sort state) to screen reader users
```

Comment:
Would it be worthwhile to link to the `announce` helper?

Reply:
Instead of saying "Best effort" can we explain the title/aria-describedby usage? Or are you trying to avoid that? Something like:

> Changes to sort order are reflected in `title` and `aria-describedby` to ensure they are announced by screen readers"

Reply:
I was trying to avoid getting too specific on the implementation details in case they changed, but I'll rethink the wording.